frontend/src/metabase/visualizations/visualizations/Table.jsx

```diff
@@ -34,7 +37,11 @@ import cx from "classnames";
 
 import { getIn } from "icepick";
 
-export default class Table extends Component {
+const mapStateToProps = state => ({
+  datasetEditorTab: "qb" in state && getDatasetEditorTab(state),
+});
```

Comment:
Why do we need to check if `qb` is in `state`? It should always be there

Reply:
There was an e2e test where this was failing because `qb` wasn't in `state`. I've replaced this anyway with prop drilling in my [latest commit](https://github.com/metabase/metabase/pull/24882/commits/2a2d502b9914bb774cd7dbc548b041c165bd031d).